test/providers/test_googlemaps.js

```diff
@@ -180,3 +180,101 @@ describe('GoogleMaps', function() {
         });
     });
 });
+
+describe('GoogleMaps.fixture', function() {
+    it('should return a fixture', function() {
+        assert.deepEqual(
+            GoogleMaps.fixture({
+                query: "Baker Street",
+                address_list: ["2B", "Not 2B"],
+            }),
+            {
+                request: {
+                    method: "GET",
+                    url: "http://maps.googleapis.com/maps/api/geocode/json",
+                    params: {
+                        address: "Baker Street",
+                    },
+                },
+                response: {
+                    code: '200',
+                    data: {
+                        status: 'OK',
+                        results: [
+                            {formatted_address: '2B'},
+                            {formatted_address: 'Not 2B'},
+                        ]
+                    },
+                },
+            }
+        );
+    });
+
+    it('should require a query parameter', function() {
+        assert.throws(
+            function() {
+                GoogleMaps.fixture({
+                    address_list: ["2B"],
+                });
+            },
+            providers.FixtureParameterMissingError
```

Comment:
Do we maybe want to assert on the error message too, just in case we end up raising this error type for something else at some point?